pkg/arvo/lib/pill.hoon

```diff
@@ -77,7 +77,7 @@
   =?  hav  ?=(^ fil.lon)
       ::  XX  this whitelist needs to be reviewed
       ::
-      ?.  ?=  ?(%css %hoon %html %js %json %md %png %txt %udon %umd)
+      ?.  ?=  ?(%css %hoon %html %js %json %md %png %txt %udon %umd %woff2)
```

Comment:
Adding these files to the pill is fine, but the same scry-format and serialization need to be applied below. And this will be a new case; we have `@t` and `$wain`,  but no $`octs`.

Reply:
TBH I don't even know what I'm doing in this file I just heard there were pngs